recipes/libarchive/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "3.2.1" %}
+
+package:
+  name: libarchive
+  version: {{ version }}
+
+source:
+  fn: libarchive-{{ version }}.tar.gz
+  url: https://github.com/libarchive/libarchive/archive/v{{ version }}.tar.gz
+  sha256: bcefe75f3f1527656c82e42e7adf1614ae274d005b2583d0ae15f857bfb1ed28
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - bzip2 1.0.*
+    - expat
+    - libiconv
+    - openssl 1.0.*
+    - xz 5.0.*
+    - zlib 1.2.*
+
+  run:
+    - bzip2 1.0.*
+    - expat
+    - libiconv
+    - openssl 1.0.*
+    - xz 5.0.*
+    - zlib 1.2.*
+
+test:
+  commands:
+    # Verify pkg-config file is in place.
+    - test -f "${PREFIX}/lib/pkgconfig/libarchive.pc"  # [unix]
+
+    # Verify headers are in place.
+    - test -f "${PREFIX}/include/archive.h"            # [unix]
+    - test -f "${PREFIX}/include/archive_entry.h"      # [unix]
+
+    # Verify libraries are in place.
+    - test -f "${PREFIX}/lib/libarchive.a"             # [unix]
+    - test -f "${PREFIX}/lib/libarchive.so"            # [linux]
+    - test -f "${PREFIX}/lib/libarchive.dylib"         # [osx]
+
+    # Check for commands
+    - bsdcat --version
+    - bsdcpio --version
+    - bsdtar --version
+
+about:
+  home: http://www.libarchive.org/
+  summary: Multi-format archive and compression library
+  license: BSD 2-Clause
```

Comment:
The license says most everything is under BSD 2-Clause, but there are some things under BSD 3-Clause, some that are public domain, and a few of build related files are under various licenses. However, they plan to go to BSD 2-Clause in the long run. Not sure what we want to do here.